commit 29d4273d878d4f3eb408342c5bd19134fc714f54 (refs/changes/24/4524/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-07-16T23:31:14+00:00 (1 year, 3 months ago)
    
    TRITON-598 "triton network get-default" should print error when no default network is set

diff --git a/lib/do_network/do_get_default.js b/lib/do_network/do_get_default.js
index 3340547..df848ea 100644
--- a/lib/do_network/do_get_default.js
+++ b/lib/do_network/do_get_default.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  *
  * `triton network get-default ...`
  */
@@ -47,7 +47,7 @@ function do_get_default(subcmd, opts, args, cb) {
             var defaultNetwork = conf.default_network;
 
             if (!defaultNetwork) {
-                cb('default network not found');
+                cb(new Error('account has no default network configured'));
                 return;
             }
 
